### DISCUSSION TITLE
Question: Consumer only on one broker

### DISCUSSION THREAD

**Discussion by ammannit** [arkmq-org/arkmq-org-broker-operator#990] [General]
Hello together

We have deployed the operator on k8s and have deployed two brokers. Then we connected our application via the svc: artemis-borker-hdls-svc

But we only see the consumers for one broker and they are not visible in the Management Console under "Consumer". If we call up the metrics via curl directly on the broker, we also only see the consumers for one broker. We would have expected to see the consumers of all brokers in the Management Console.

Our goal was to set up an HA (master & slave) so that if one broker fails, the other broker takes over. Apparently this is not possible with the operator. How do we ensure HA with an operator? How can we display all consumers in the Management Console? Do we understand something wrong?

Thx!

<img width="437" alt="Bildschirmfoto 2024-07-23 um 21 59 40" src="https://github.com/user-attachments/assets/046812ae-197c-43c9-b062-9233673411ea">
<img width="586" alt="Bildschirmfoto 2024-07-23 um 21 59 33" src="https://github.com/user-attachments/assets/08ce5a28-1976-4138-8542-b49a40c77247">

**Comment by brusdev** [comment]
The ArtemisCloud Operator deploys a cluster of primary brokers and each broker only show its clients. You could use the Prometheus Operator to aggregate metrics from multiple broker instances.

You can find an example of JDBC HA at https://github.com/artemiscloud/activemq-artemis-operator/blob/1.2.4/controllers/activemqartemis_jdbc_ha_test.go
